File: SoftUni/Data Structures - September 2020/Linear Data Structures/Problem04.SinglyLinkedList/Node.cs
namespace Problem04.SinglyLinkedList
{
    public class Node<T>
    {
        public T value;

        public Node<T> next;

        public Node(T value, Node<T> next = null)
        {
            this.value = value;
            this.next = next;
        }
    }
}

File: Java/Internship Spring/ADS/src/main/java/com/Helevator/ads/queues/LinkedQueue/Node.java
package com.Helevator.ads.queues.LinkedQueue;

public class Node {
    public int Data;
    Node Next;

    public Node() {
        this(0);
    }

    public Node(Integer data) {
        this.Data = data;
    }

    public Node(Integer data, Node next) {
        this.Data = data;
        this.Next = next;
    }
}
